apps/opendosm/dashboards/formal-sector-wages/index.tsx

```diff
@@ -0,0 +1,227 @@
+
+  const STATES_OPTION: Array<OptionType> = [MALAYSIA, ...STATES].map(state => ({
+    label: state.name,
+    value: state.key,
+  }));
+
+  const { data, setData } = useData({
+    minmax: [0, timeseries.data[STATES_OPTION[0].value]?.x.length - 1],
+    state: STATES_OPTION[0].value,
+    barChart_index: barBracketDates.length - 1,
+  });
+  const LATEST_TIMESTAMP = timeseries.data[data.state].x[timeseries.data[data.state].x.length - 1];
+  const { coordinate } = useSlice(timeseries.data[data.state], data.minmax);
+
+  return (
+    <>
+      <Hero
+        background="purple"
+        category={[t("common:categories.labour_markets"), "text-purple"]}
+        header={[t("header")]}
+        description={[t("description")]}
+        last_updated={last_updated}
+        agencyBadge={<AgencyBadge name={t("agencies:mbls.full")} icon={<JIMIcon />} />}
+      />
+
+      <Container className="min-h-screen">
+        <Section title={t("section_barchart.title")} date={bar_bracket.data_as_of}>
+          <div className="grid w-full grid-cols-1 gap-12 xl:grid-cols-2">
+            <Bar
+              className="h-[550px]"
+              title={t("keys.bar_bracket_title")}
+              layout="horizontal"
+              reverse={true}
+              data={{
+                labels: barBracketItem[data.barChart_index].x,
+                datasets: [
+                  {
+                    label: t("keys.bracket"),
+                    data: barBracketItem[data.barChart_index].y,
+                    barThickness: 25,
+                    borderWidth: 0.5,
+                    borderColor: theme === "light" ? AKSARA_COLOR.PURPLE : AKSARA_COLOR.BLACK,
+                    backgroundColor:
+                      theme === "light" ? AKSARA_COLOR.PURPLE_H : AKSARA_COLOR.PURPLE,
+                    hoverBackgroundColor:
+                      theme === "light" ? AKSARA_COLOR.PURPLE : AKSARA_COLOR.PURPLE_H,
+                  },
+                ],
+              }}
+              enableGridX={true}
+            />
+
+            <Bar
+              className="h-[550px]"
+              title={t("keys.bar_percentile_title")}
+              layout="horizontal"
+              data={{
+                labels: barPercentileItem[data.barChart_index]?.x,
+                datasets: [
+                  {
+                    label: t("keys.percentile"),
+                    data: barPercentileItem[data.barChart_index]?.y,
+                    barThickness: 25,
+                    borderWidth: 0.5,
+                    borderColor: theme === "light" ? AKSARA_COLOR.PURPLE : AKSARA_COLOR.BLACK,
+                    backgroundColor:
+                      theme === "light" ? AKSARA_COLOR.PURPLE_H : AKSARA_COLOR.PURPLE,
+                    hoverBackgroundColor:
+                      theme === "light" ? AKSARA_COLOR.PURPLE : AKSARA_COLOR.PURPLE_H,
+                  },
+                ],
+              }}
+              enableGridX={true}
+            />
```

Comment:
Maybe enableGridY = false ? Get rid of the horizontal lines on chart, but keep the vertical ones